recipe/meta.yaml

```diff
@@ -25,20 +24,21 @@ requirements:
     - numpy   1.20  # [not (osx and arm64) and py<310]
```

Comment:
cython: https://github.com/alkaline-ml/pmdarima/blob/v2.0.3/requirements.txt#L2

Reply:
cython is already set, at line 22...

Reply:
it's missing `,!=0.29.31`